app/classifier/tasks/mobile-section.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import AutoSave from '../../components/auto-save';
+
+export default class MobileSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.toggleSwipeEnabled = this.toggleSwipeEnabled.bind(this);
+  }
+
+  toggleSwipeEnabled(e) {
+    const changes = {};
+    changes['configuration.swipe_enabled'] = e.target.checked;
+    this.props.workflow.update(changes).save();
+    if (e.target.checked) {
+      this.props.project.update({ mobile_friendly: e.target.checked }).save();
```

Comment:
Note: This part of the code also sets the project to `mobile_friendly: true` if not currently active.